src/components/Maincontainer.css

```diff
@@ -1,275 +1,31 @@
-h1,
-h2,
-h3,
-h4,
-h5,
-h6 {
-  font-weight: 1000;
-  font-family: "Baloo Paaji", cursive;
-  color: #695aa6;
+@media only screen and (max-width: 767px) {
+  .hlBpIo {
```

Comment:
This is auto-generated class name from React algorithm. Never use this, Instead give this element a class name and then it can be used like this.

Reply:
The values `11%`and `3%` doesn't seem proper.

Reply:
Done